CHANGELOG.md

```diff
@@ -5,6 +5,12 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
+## [1.5.2] - 2020-12-15
```

Comment:
If aligning to SemVer, would suggest adding a new module would be a minor change (new feature, backward compatiable, non-breaking), and so would suggest bump to 1.6.0.  Third digit normally reserved for bug fixes.

Reply:
@adamdmharvey good point, change made